salesagility/SuiteCRM
# Partially fixed #3756 - Calendar pop-ups now auto close after 500ms w…

…hen the mouse leaves the event or the popup itself



## Description



See issue: #3756 

## Motivation and Context

Now, pop-ups don't auto-close, cluttering the Calendar screen.

## How To Test This

Create an event that shows up on the Calendar. GO to the Calendar, hover your mose over the event and see if the pop-up auto-closes 500ms after mouseout.

## Types of changes


### Final checklist


- [n.a.] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
